### PR TITLE
Stress tester - WIP

### PR DESCRIPTION
**Pull request by vDrWool**
Performance Tester helps us to check for some 'jump' in performance.
It creates some instances of app that we need to check and pass from 25 to 75 args (like 'cl', 'mh', etc).

Unfortunatly, for now **perfTest is working via 'crutch'**. I added a delay of 5 seconds for the client, otherwise the client crashes when passing arguments to it. **This have to be solved somehow.**

And also I need that _someone check it on Linux and MacOS_.

### CODE REVIEW

**Comment by Svidunovich** [comment]
Do we need all the comments in this pr?

**Comment by vDrWool** [comment]
> Do we need all the comments in this pr?
> If not, fix it
> plz

**Comment by vDrWool** [comment]
Comments in Tester.cpp I won't to delete becase they are about how it works (I guess it a bad idea).

File: PerformanceTest/ArgumentParser.cpp [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	{
+		switch (arg)
+		{
+		case argument::path:
+			return fs::exists(value);
+		case argument::count:
+			return !value.empty()
+				&& (std::atoi(value.data()) < _MAX_COUNT);
+				// WHY IS IT NOT COMPILING!??!!?
+				//&& std::all_of(value.cbegin(), value.cend(), std::isdigit);
```
Comment (Svidunovich) [inline PerformanceTest/ArgumentParser.cpp:27]: Do we need these comments?

File: PerformanceTest/SimpleTimer.cpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+	void SimpleTimer::stop() noexcept 
+	{ 
+		_end = hsc::now();
+	}
+
+	void SimpleTimer::reset() noexcept 
+	{ 
+		_start        = {};
+        _end          = {};
```
Comment (Svidunovich) [inline PerformanceTest/SimpleTimer.cpp:20]: Please check spaces in all documents

**Review by Svidunovich — CHANGES_REQUESTED** [review]

**Review by Svidunovich — APPROVED** [review]

**Review by StasDash — APPROVED** [review]

**Review by GuskoMaksim — APPROVED** [review]

**Pull request merged**